tracer/build/_build/Build.ExplorationTests.cs

```diff
@@ -0,0 +1,252 @@
+using System;
+using System.IO;
+using System.Linq;
+using Nuke.Common;
+using Nuke.Common.IO;
+using Nuke.Common.Tools.DotNet;
+using Nuke.Common.Tooling;
+using Nuke.Common.Tools.Git;
+using static Nuke.Common.Tools.DotNet.DotNetTasks;
+
+public enum MonitoringType
+{
+    Debugger, Profiler
```

Comment:
```suggestion
    Debugger, ContinuousProfiler
```

The word "Profiler" alone is too overloaded in this context :)

Reply:
Renamed to "ContinuousProfiler"